### PR TITLE
Possible fix for #79

### PR DESCRIPTION
**Pull request by acoll**
Issue #79 

I just added a simple truthy check to prevent the error. I'm not sure this is the right fix or not.

### CODE REVIEW

**Comment by circlingthesun** [comment]
Works for me :+1:

**Comment by circlingthesun** [comment]
Works on both Webpack 3.5.0 and 4.0.0 for me.

**Comment by sentilesdal** [comment]
Worked for me too!

**Comment by KnisterPeter** [comment]
@bholloway Works fine. `this.options` is just not set for webpack 4 anymore

File: index.js [hunk 1]
```diff
@@ -36,7 +36,10 @@ function resolveUrlLoader(content, sourceMap) {
 
   // webpack 1: prefer loader query, else options object
   // webpack 2; prefer loader options
-  var options = defaults(loaderUtils.getOptions(loader), loader.options[camelcase(PACKAGE_NAME)], {
+  var options = defaults(
+      loaderUtils.getOptions(loader), 
+      loader.options && loader.options[camelcase(PACKAGE_NAME)], 
+      loader.query, {
```
Comment (bholloway) [inline index.js]: @acoll It seems to me that `loader.query` is redundant given that `loaderUtils.getOptions(loader)` incorporates it [here](https://github.com/webpack/loader-utils#getoptions).

**Comment by bholloway** [comment]
@acoll Don't panic if you see "closed" on the PR, I merged this correctly via the git command line per the github instructions.

**Comment by acoll** [comment]
👍 sounds good.  Thanks for fixing that.

**Pull request closed without merging**